intro_to_python/intro_to_python.md

```diff
@@ -2,21 +2,22 @@
 
 author:   Meredith Lee
 email:    leemc@chop.edu
-version:  1.0.0
+version:  2.0.0
+module_template_version: 2.0.0
 language: en
 narrator: UK English Female
 title: Introduction to Python
```

Comment:
Should we consider renaming this module getting started with python, getting set up with python or something like that? (I know annoyingly that would require creating new folders and lots of renaming stuff)

I'm just wondering if Introduction to Python still feels like the most descriptive title now that the scope of the module has been limited so much, and much of the other introductory content will be going in other places. 

There was definitely a point in my own learning curve where I already had anaconda installed, but still would have been searching for "introduction to python" content. 

As long as the comment + long description are accurate, this shouldn't matter too too much, but I think still at least worth considering!

Reply:
I think you're right, it could be a bit more descriptive... I think I like "Getting started with Python"... let's take one last look at the content before I make that change though, since changing file names messes with the change log.